test/actions.test.js

```diff
@@ -0,0 +1,60 @@
+import * as moviesActions from '../src/actions'; 
+
+const movieData = {data:[{
+    "id": 337167,
+    "title": "Fifty Shades Freed",
+    "tagline": "Don't miss the climax",
+    "vote_average": 6.1,
+    "vote_count": 1195,
+    "release_date": "2018-02-07",
+    "poster_path": "https://image.tmdb.org/t/p/w500/3kcEGnYBHDeqmdYf8ZRbKdfmlUy.jpg",
+    "overview": "Believing they have left behind shadowy figures from their past, newlyweds Christian and Ana fully embrace an inextricable connection and shared life of luxury. But just as she steps into her role as Mrs. Grey and he relaxes into an unfamiliar stability, new threats could jeopardize their happy ending before it even begins.",
+    "budget": 55000000,
+    "revenue": 136906000,
+    "genres": [
+    "Drama",
+    "Romance"
+    ],
+    "runtime": 106
+}]};
+
+describe('moviesActions', () => {
+    test('should create action to get movies', () => {
+        const expectedAction = {
+            type: moviesActions.RECEIVE_MOVIES,
+            movies: movieData.data
+        }
+
+        expect(moviesActions.receiveMovies(movieData)).toEqual(expectedAction);
+    });
```

Comment:
Tests should be as simple as possible. And mock data should contain only significant data. Having this test written as below, `movieData` can be removed.

```js
    test('should create action to get movies', () => {
        const moviesMock = {
          data: 'movie_data',
        };

        const expectedAction = {
            type: moviesActions.RECEIVE_MOVIES,
            movies: 'movie_data'
        }

        expect(moviesActions.receiveMovies(moviesMock )).toEqual(expectedAction);
    });
```